schemes/check_energy/check_energy_chng.meta

```diff
@@ -0,0 +1,407 @@
+  intent = in
+[ T ]
+  standard_name = air_temperature
+  units = K
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension, vertical_layer_dimension)
+  intent = in
+[ pintdry ]
+  standard_name = air_pressure_of_dry_air_at_interface
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension, vertical_interface_dimension)
+  intent = in
+[ phis ]
+  standard_name = surface_geopotential
+  units = m2 s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = in
+[ zm ]
+  standard_name = geopotential_height_wrt_surface
+  units = m
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension, vertical_layer_dimension)
+  intent = in
+[ cp_phys ]
+  standard_name = composition_dependent_specific_heat_of_dry_air_at_constant_pressure
+  units = J kg-1 K-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension, vertical_layer_dimension)
+  intent = in
+[ cp_or_cv_dycore ]
+  standard_name = specific_heat_for_air_used_in_dycore
+  units = J kg-1 K-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension, vertical_layer_dimension)
+  intent = in
+[ te_ini_phys ]
+  standard_name = vertically_integrated_total_energy_of_initial_state_using_physics_energy_formula
+  units = J m-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = inout
+[ te_ini_dyn ]
+  standard_name = vertically_integrated_total_energy_of_initial_state_using_dycore_energy_formula
+  units = J m-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = inout
+[ tw_ini ]
+  standard_name = vertically_integrated_water_vapor_and_condensed_water_of_initial_state
+  units = kg m-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = inout
+[ te_cur_phys ]
+  standard_name = vertically_integrated_total_energy_of_current_state_using_physics_energy_formula
+  units = J m-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = inout
+[ te_cur_dyn ]
+  standard_name = vertically_integrated_total_energy_of_current_state_using_dycore_energy_formula
+  units = J m-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = inout
+[ tw_cur ]
+  standard_name = vertically_integrated_water_vapor_and_condensed_water_of_current_state
+  units = kg m-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = inout
+[ tend_te_tnd ]
+  standard_name = cumulative_total_energy_boundary_flux_using_physics_energy_formula
+  units = J m-2 s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = out
+[ tend_tw_tnd ]
+  standard_name = cumulative_total_water_boundary_flux
+  units = kg m-2 s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension)
+  intent = out
+[ temp_ini ]
+  standard_name = temperature_of_initial_state
```

Comment:
We should probably specify "air" here:

```suggestion
  standard_name = air_temperature_of_initial_state
```

Reply:
Thanks, agreed, will wait to change this until we decide on `_initial_state`

Reply:
Updated to `air_temperature_at_start_of_physics_timestep`